boards.txt

```diff
@@ -11383,6 +11383,105 @@ heltec_wifi_kit_32.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+heltec_wifi_kit_32_V3.name=heltec WiFi Kit 32(V3)
+
+heltec_wifi_kit_32_V3.vid.0=0x303a
+heltec_wifi_kit_32_V3.pid.0=0x1001
+
+heltec_wifi_kit_32_V3.bootloader.tool=esptool_py
+heltec_wifi_kit_32_V3.bootloader.tool.default=esptool_py
+
+heltec_wifi_kit_32_V3.upload.tool=esptool_py
+heltec_wifi_kit_32_V3.upload.tool.default=esptool_py
+heltec_wifi_kit_32_V3.upload.tool.network=esp_ota
+
+heltec_wifi_kit_32_V3.upload.maximum_size=1310720
+heltec_wifi_kit_32_V3.upload.maximum_data_size=327680
```

Comment:
When partition is fixed to default_8MB, I can't see that upload.maximus_size corresponds to the partition default_8MB.

`esp32s3.menu.PartitionScheme.default_8MB.upload.maximum_size=3342336`

Reply:
dsl I did not understand what to do

Reply:
I mean, when this board definition is always set to 8MB Flash size, the maximum upload size should be different than the default. You use partition scheme default_8MB so the `heltec_wifi_kit_32_V3.upload.maximum_size=1310720` should be `3342336`.

Reply:
Do know what I mean by this? I see no change.